### ISSUE TITLE
"Date taken" broken after copying images to new SD card?

### ISSUE THREAD

**Issue by kr428** [SimpleMobileTools/Simple-Gallery#2142]
Unsure how to fix this: Early this month I replaced the SD card in my phone and copied over all the images. While this works for most, for roughly half of the files on my device (9000 images) apparently sorting or grouping by "Date Taken" doesn't work anymore. Looking to the info panel in the app, it seems to correctly display "Date taken" (as the original date this image was taken on, which corresponds with the file name on my device) and "last modification" (which is the date on which I copied these files to the new SD card).
Following the FAQ on your website, I tried to fix these dates manually but this makes things only worse - as it doesn't use the "Date taken" stored with the image and use this but instead copies the "last modification" date to both "Date taken" field and image file name. This is quite odd. Any way to track down why this is happening? Sorting/grouping by Date taken was one core feature why I started using Simple Gallery but this is pretty much unusable at the moment. :(

**Comment by tibbi** [comment]
What app version are you using?

**Comment by kr428** [comment]
Latest Pro version from Play Store.

**Comment by tibbi** [comment]
"Fix Date Taken values" should take the date you see at the Properties dialog too. Do other apps sort images well?

**Comment by kr428** [comment]
Yes, checked with some other gallery apps as well as file manager  (that is capable of sorting by date taken, too). I also do not really see a difference between pictures that are sorted "well" and those that aren't. It seems they all do have "Date/Time", "Date/Time Original" and "Date Time Digitized" set to the date when the image was captured, and sorting works well in all other apps. Simple Gallery is capable of displaying the right date but somehow(?) mixes this with the date of last modification...

**Comment by tibbi** [comment]
Try finding the app in your device settings and do a "Clear data" (or "Clear storage") on it. It will just reset your app settings and clear the recycle bin/favorites, wont delete any files. Then fix date taken again.

**Comment by kr428** [comment]
Already did that because I thought something in cache was messed up - to the point of removing and re-installing the app, no change. :(

**Comment by tibbi** [comment]
I dont see how can it still not work. Arent you confusing sorting and grouping?

**Comment by tibbi** [comment]
try disabling grouping at the top Menu -> Group by

**Comment by kr428** [comment]
Already tried that. Played with various combinations of sorting and grouping. No real change. :( Is there any way to debug what's happening?

**Comment by tibbi** [comment]
not really, so far nobody had such an issue. Maybe if I send you a test version with a message appearing I could check what value is the Date Taken stored under, maybe the app cannot parse it properly and thats why Fix Date Taken doesnt work. Let me know at hello@simplemobiletools.com if you want that, it is easier to send files there.
